src/inmanta/env.py

```diff
@@ -508,6 +511,7 @@ def create_log_content_files(title: str, files: list[str]) -> list[str]:
             not_found: list[str] = []
             conflicts: list[str] = []
             indexes: str = ""
+            no_index: bool = "--no-index" in cmd or strtobool(env.get("PIP_NO_INDEX", "false").lower())
```

Comment:
The `strtobool()` method already does the conversion to `lower()`.
```suggestion
            no_index: bool = "--no-index" in cmd or strtobool(env.get("PIP_NO_INDEX", "false"))
```